src/js/tabs/osd.js

```diff
@@ -988,6 +1005,7 @@ OSD.chooseFields = function () {
     OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
       F.FLYMODE,
       F.CRAFT_NAME,
+      F.DISPLAY_NAME,
```

Comment:
This will have to be added below, in the `if (semver.gte(CONFIG.apiVersion, "1.41.0")) {` block.